providers/gitlab.go

```diff
@@ -173,7 +173,7 @@ func (p *GitLabProvider) verifyGroupMembership(userInfo *gitlabUserInfo) error {
 	}
 
 	// Find a valid group that they are a member of
-	validGroups := strings.Split(p.Group, " ")
+	validGroups := p.Group
```

Comment:
I don't think we need this variable do we, can we just inline it on the next line?